microsoft-edge/devtools-guide-chromium/network/index.md

```diff
@@ -220,30 +238,34 @@ Select a resource to learn more information about it.
 
 <!-- ====================================================================== -->
 ## Search network headers and responses
+<!-- https://developer.chrome.com/docs/devtools/network#search -->
 
 Use the **Search** pane when you need to search the HTTP headers and responses of all resources for a certain string or regular expression.
 
-For example, suppose you want to verify that your resources are using reasonable **cache policies**.<!--[cache policies](../../../web/tools/lighthouse/audits/cache-policy) -->
-
-<!--TODO: add cache policies section when available  -->
+For example, suppose you want to verify that your resources are using reasonable cache policies.<!-- todo: add & link to cache policies section: [cache policies](https://web.dev/uses-long-cache-ttl/) ends up at [Serve static assets with an efficient cache policy](https://developer.chrome.com/docs/lighthouse/performance/uses-long-cache-ttl) -->
```

Comment:
rewrote TODO comment, updated hidden link(s).  Upstream's link redirects.
Further Maint work here is likely out of scope for this PR.
@captainbrosset

Reply:
I think we can link to this MDN guide on HTTP Caching: https://developer.mozilla.org/en-US/docs/Web/HTTP/Guides/Caching

```suggestion
For example, use the **Search** pane to verify that your resources are using the correct cache policies. To learn more about HTTP caching, see [HTTP caching](https://developer.mozilla.org/docs/Web/HTTP/Guides/Caching).
```

No need to go too much into the details of what efficient caching is here, let's not be too clever. We can't assume what readers will use the search tool for.

Reply:
result:
```
For example, use the **Search** pane to verify that your resources are using the correct cache policies, as follows:
...

See also:
* [HTTP caching](https://developer.mozilla.org/docs/Web/HTTP/Guides/Caching) at MDN.

```